taskcluster/rb_taskgraph/transforms/secrets.py

```diff
@@ -2,8 +2,7 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 """
-Apply some defaults and minor modifications to the jobs defined in the nimbledroid
-kind.
+Resolve secrets and dummy secrets
```

Comment:
Out of scope for this PR, but I'd consider moving this logic into the job types that use them, rather than having an additional transform for them. (I'd be less inclined to do so if more values where passed beyond `level` though)

Reply:
Makes sense. I'll get this done in a followup. Thank you for pointing it out :)